installer/lib/phoenix_new.ex

```diff
@@ -31,6 +31,11 @@ defmodule Mix.Tasks.Phoenix.New do
     {:eex,  "new/web/web.ex",                                "web/web.ex"},
     {:eex,  "new/mix.exs",                                   "mix.exs"},
     {:eex,  "new/README.md",                                 "README.md"},
+    {:eex,  "new/web/gettext.ex",                            "web/gettext.ex"},
+    {:eex,  "new/priv/gettext/default.pot",                  "priv/gettext/default.pot"},
```

Comment:
Let's not add the default files. Let's let the user generate those ones. Specially because the initial message we translate will likely be removed anyway.